### PR TITLE
Truly fixed modtools

### PR DESCRIPTION
**Pull request by matthew-morrison**
Every function was tested this time.

### CODE REVIEW

File: modtools.py [hunk 1]
```diff
@@ -12,18 +12,20 @@ def __init__(self, bot):
 
     @commands.command(pass_context=True)
     async def lock(self, ctx):
-        if Minion(ctx):
+        """Locks the current channel."""
+        if self.minion(ctx):
             everyone = ctx.message.server.default_role
-            await ch_perms(ctx.message.channel, everyone, False)
+            await self.ch_perms(ctx.message.channel, everyone, False)
```
Comment (henrymzhao) [inline modtools.py]: if you set everyone to false, then only I can unlock a channel after it's locked?

Comment (matthew-morrison) [inline modtools.py]: hmmmmm if a person has a role which allows them to explicity speak in a channel, then they can continue to speak. However, this has not been extensively tested.

File: modtools.py [hunk 2]
```diff
@@ -67,11 +69,11 @@ def __init__(self, bot):
...
                         await self.bot.say("You ain't no mod, shoo!")
 
-    def Minion(ctx):
-        if "&314296819272122368" in ctx.message.author.roles.id: # >.>
-            return True
-        else:
-            return False
+    def minion(self, ctx):
+        for role in ctx.message.author.roles:
+            if "314296819272122368" == role.id:
```
Comment (henrymzhao) [inline modtools.py]: are you sure the `&` sign isn't part of the ID?

File: modtools.py (hunk 1 above)
Comment (henrymzhao) [inline modtools.py]: so basically nobody but me lol.

File: modtools.py (hunk 2 above)
Comment (matthew-morrison) [inline modtools.py]: Yes, role.id prints out as a string of digits.

File: modtools.py (hunk 1 above)
Comment (matthew-morrison) [inline modtools.py]: Moderators who can edit channel permissions can unlock a channel manually by editing the @everyone role in a specific channel.

Comment (henrymzhao) [inline modtools.py]: Ah. Can you then add a modifier so it mutes **everyone**.

ie. 
`.lock //locks channel`
`.lock all //locks **everyone** including moderators`

you know, for those pesky "moderator" fights.

**Pull request merged**